portal/views/pages/address-detail.njk

```diff
@@ -7,15 +7,16 @@
 {% block main %}
   <h1 class="main-heading">{{ _('section.addressOverview') }}</h1>
   {% include 'partials/address/address-info.njk' %}
+  <hr />
   {% include 'partials/address/address-tokens.njk' %}
 
-  <div class="block-section">
+  <div class="block-section is-mobile address-detail-page">
```

Comment:
Hi, just wondering, I see the `is-mobile` class is added to this div, will that make these code 
```
<h2>{{ _('navbar.transactions') }} ({{ total | numberFormat }} {{ _('helper.field.total') }})</h2>
```
 to be not displayed on desktop?

Reply:
Remove all duplicates of is-mobile